jekyll/_ccie/gpu-configuration.md

```diff
@@ -206,4 +206,4 @@ Then in your circle.yml test section, you'll need to add
     - export LD_LIBRARY_PATH=/usr/local/cuda-8.0/lib64:$LD_LIBRARY_PATH
 
     
-Now you'll have GPU access in your products!
\ No newline at end of file
+Now you'll have GPU access in your products!
```

Comment:
I have no idea. That's really strange. I didn't change anything on that line and can't see any differences in the diff either.